examples/remix-app-node-vm/README.md

```diff
@@ -17,6 +17,4 @@ Some key aspects to note regarding this example are:
   - [creating the ssr environment](https://github.com/dario-piotrowicz/remix/blob/6a8c12380453c7fd01718810683475ccdb690eff/packages/remix-dev/vite/plugin.ts#L1049-L1065)
   - [creating a handler using the dev environment](https://github.com/dario-piotrowicz/remix/blob/6a8c12380453c7fd01718810683475ccdb690eff/packages/remix-dev/vite/plugin.ts#L1427-L1431)
 
-- note that in the remix plugin we specified some options for the ssr environment in case the the node-vm runtime is used (see [here](https://github.com/dario-piotrowicz/remix/blob/6a8c12380453c7fd01718810683475ccdb690eff/packages/remix-dev/vite/plugin.ts#L1333-L1356)) those are `mainFields` and `external`:
-- `mainField` is needed to make sure that the correct module resolution is applied, we should address this, but for now we're simply patching Vite (for more context see our [patch](../../patches/vite@6.0.0-alpha.18.patch) and this [discussion](https://github.com/vitejs/vite/pull/16471#discussion_r1619160848))
-- `external` is simply taken from the existing plugin's ssr configuration: [source](https://github.com/dario-piotrowicz/remix/blob/da96d35cf6a01578582d125ec4cb979bdb4a74f8/packages/remix-dev/vite/plugin.ts#L1056-L1077) so we're not adding this.
```

Comment:
I've removed this when I removed the patch, also setting `mainField` is no longer necessary without the patch